{"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/27/comments/45633126.json"}, "html": {"href": "#!/levitsky/pyteomics/issues/27#comment-45633126"}}, "issue": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics/issues/27.json"}}, "type": "issue", "id": 27, "repository": {"links": {"self": {"href": "data/repositories/levitsky/pyteomics.json"}, "html": {"href": "#!/levitsky/pyteomics"}, "avatar": {"href": "data/bytebucket.org/ravatar/{3f6302de-10da-4d3c-b258-0bb6a2368972}ts=74456"}}, "type": "repository", "name": "pyteomics", "full_name": "levitsky/pyteomics", "uuid": "{3f6302de-10da-4d3c-b258-0bb6a2368972}"}, "title": "`XML._retrieve_refs` is undefined"}, "content": {"raw": "Sadly, there are references in mzML too.  I was prototyping support for `referenceableParamGroupRef` tags in my mzML writer now and noticed that passing `retrieve_refs=True` to `MzML.iterfind` threw an `AttributeError`.", "markup": "markdown", "html": "<p>Sadly, there are references in mzML too.  I was prototyping support for <code>referenceableParamGroupRef</code> tags in my mzML writer now and noticed that passing <code>retrieve_refs=True</code> to <code>MzML.iterfind</code> threw an <code>AttributeError</code>.</p>", "type": "rendered"}, "created_on": "2018-05-21T13:54:24.338660+00:00", "user": {"display_name": "Joshua Klein", "uuid": "{919f0add-304d-4b9a-8889-d2622a3dbc96}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B919f0add-304d-4b9a-8889-d2622a3dbc96%7D"}, "html": {"href": "https://bitbucket.org/%7B919f0add-304d-4b9a-8889-d2622a3dbc96%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/7d0e70bc74f783efa621a2bdd228ca22d=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsJK-3.png"}}, "nickname": "mobiusklein", "type": "user", "account_id": "557058:ff82222f-afe5-4135-a1b7-8de99a00f669"}, "updated_on": null, "type": "issue_comment", "id": 45633126}